ci/infrastructure/modules/cluster/kubernetes/install.tf

```diff
@@ -17,7 +17,7 @@ resource "null_resource" "kubernetes-install-master" {
   provisioner "remote-exec" {
     inline = [
       "sudo kubeadm init --ignore-preflight-errors=swap,cri --apiserver-bind-port ${local.master_port} --kubernetes-version ${var.kube_version} --token `cat kubernetes.token` --pod-network-cidr=192.168.0.0/16 --apiserver-cert-extra-sans=${join(",", concat(var.master_public_ips, var.master_public_dns_names))}",
-      "${local.pod_network_112_plus == "true" ? "sudo KUBECONFIG=/etc/kubernetes/admin.conf kubectl apply -f https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml; sudo KUBECONFIG=/etc/kubernetes/admin.conf kubectl apply -f https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml" : "sudo KUBECONFIG=/etc/kubernetes/admin.conf kubectl apply -f https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml"}",
+      "${local.pod_network_112_plus == "true" ? "sudo KUBECONFIG=/etc/kubernetes/admin.conf kubectl apply -f https://docs.projectcalico.org/v${local.kube_calico_version}/manifests/calico.yaml" : "sudo KUBECONFIG=/etc/kubernetes/admin.conf kubectl apply -f https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml"}",
```

Comment:
This implementation is a bit hard to understand.  Correct if I'm wrong, currently this states:
 

if k8s > 1.11, use calico 3.10, otherwise use `https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml`.

What do you need the `kube_calico_version` variable for?

Reply:
it is just a variable. we can simply change the variable for different versions. For Kube version less than 11. I have fixed the value to 3.9.